devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -223,3 +254,77 @@ private getModelBindings(model) {
 	}
 	bindings
 }
+
+private Map parseAduroSmartButtonMessage(Map descMap){
+    def buttonState = ""
+    def buttonNumber = 0
+    if (descMap.clusterInt == 0x0006) {
```

Comment:
Please use constants instead of magic numbers for clusters.

0x0006 is `zigbee.ONOFF_CLUSTER`
0x0008 is `zigbee.LEVEL_CONTROL_CLUSTER`

for 0xFCCC define a function 

`def getMANUFACTURER_SPECIFIC_CLUSTER() {0xFCCC}`